pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2005,6 +2005,8 @@ func (c *Cloud) AttachDisk(diskName KubernetesVolumeID, nodeName types.NodeName)
 	// See http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html
 	ec2Device := "/dev/xvd" + string(mountDevice)
 
+	requestTime := time.Now()
```

Comment:
IDK if it's useful at all to report a metric if the disk is already attached and we don't actually call AttachVolume. @gnufied ?